linkchecker.yml

```
@@ -0,0 +1,14 @@
+on:
+  schedule:
+    - cron: '0 8 1 * *'  # Runs on the 1st of every month at 8AM GMT
```

It wouldn't hurt to run this more frequently. If we want to run it weekly on Sunday morning:

```suggestion
    - cron: '0 8 * * 0'  # Runs every Sunday at 8AM GMT
```

---

https://crontab.guru/#0_8_*_*_0 and `man 5 crontab` in case any of that is confusing.

---

Agreed, weekly runs would be good.